src/sql/workbench/parts/connection/common/serverInfoContextKey.ts

```diff
@@ -10,25 +10,35 @@ export class ServerInfoContextKey implements IContextKey<ServerInfo> {
 
 	static ServerInfo = new RawContextKey<ServerInfo>('serverInfo', undefined);
 	static ServerMajorVersion = new RawContextKey<string>('serverMajorVersion', undefined);
+	static IsCloud = new RawContextKey<boolean>('isCloud', undefined);
```

Comment:
Wait... we define these in the MSSQL extension. @YurongHe can you help advice on this? How should Gene handle this - is it OK to put in core, or does he need to send some even that lets the MSSQL extension update these values for a context?

Reply:
mssql extension loads up those context keys when dashboard opens. The context key values are from core. Do you want me to load the keys when connecting instead of opening dashboard?